capture_gui/app.py

```diff
@@ -556,8 +556,9 @@ def _read_widget_configuration(self):
         """Read the stored widget inputs"""
         inputs = {}
 
-        if not os.path.isfile(self.settingfile) or \
-                        os.stat(self.settingfile).st_size == 0:
+        is_file = os.path.isfile(self.settingfile)
+        file_size = os.stat(self.settingfile).st_size
```

Comment:
This will error out when the file does not exist, because it will now _always_ call `os.stat`. Because in the original if statement with `or` it would never call `os.stat` if the first one already made the if statement pass (when `not os.path.isfile(self.settingfile)`).

Similar to:
```python
def a():
    print 'a'
    return True
        
def b():
    print 'b'
    return True
    
if a() or b():
    print "c"
# a
# c
```
Function `b()` is never called.